packages/access-decision-manager-express/README.md

```diff
@@ -0,0 +1,151 @@
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)    [![build](https://img.shields.io/travis/wizeline/access-decision-manager/master.svg)](https://travis-ci.org/wizeline/access-decision-manager)
+
+
+ # Access-decision-manager-express.
+
+
+ Determines whether or not an entity is entitled to perform a certain action,
+for example, request a certain URI or modify a certain object. It relies on
+[access-decision-manager](https://github.com/wizeline/access-decision-manager/tree/master/packages/access-decision-manager#readme), and to make a positive verdict, it is sufficient to have the approval
+of only one Voter. Voters are implemented by the client service.
+
+
+
+ ### Implementing Voters
+
+ Voters are to determine access rights
+The use of Voters to verify permissions is based on [this model](https://symfony.com/doc/current/security/voters.html).
+Voters are called to grant access to a resource or an action.
+
+
+
+
+  Adding a Custom Voter.
+
+
+  To create a Voter,
+use the `Voter` interface:
+
+ ```typescript
+export interface Voter {
+  supports: (attribute: any, subject: any, context: any) => boolean;
+  voteOnAttribute: (
+    attribute: any,
+    subject: any,
+    user: any,
+    context: any,
+  ) => boolean | Promise<boolean>;
+}
+```
+
+
+ For example, the following voter checks attributes related to a public resources and private ones:
+
+ ```typescript
+import { Voter } from '@wizeline/access-decision-manager';
+
+const supportedAttributes = [
+  'GET_PUBLIC'
+];
+
+const publicVoter = (
```

Comment:
Similar to https://github.com/wizeline/access-decision-manager/pull/9/files#r298277181, let's simplify this since we don't need any options yet.